### DISCUSSION TITLE
월드 제작시 캐릭터 애니메이션은 어떻게 사용해야 하는지요?

### DISCUSSION THREAD

**Discussion by Yarore** [naverz/zepeto-studio-kor#68] [4. General]
월드 제작시 캐릭터 애니메이션은 어떻게 사용해야 하는지요?

템플릿에 있는 애니메이터만 사용해야하는건가요? 아니면

직접 만든 애니메이션도 월드에 넣는다면 사용할수 있는건가요?

**Comment by x4cf** [comment]
직접만든 애니메이션도 월드에 넣어 사용할 수 있습니다. 기존의 유니티와 동일하게 제작하시면 됩니다.
